source/extensions/quic_listeners/quiche/platform/quic_test_impl.h

```diff
@@ -18,3 +22,8 @@ class ScopedEnvironmentForThreadsImpl {};
 using QuicTestImpl = ::testing::Test;
 
 template <class T> using QuicTestWithParamImpl = ::testing::TestWithParam<T>;
+
+inline std::string QuicGetTestMemoryCachePathImpl() {
+  PANIC("Not implemented"); // TODO(mpwarres): implement
```

Comment:
nit: NOT_IMPLEMENTED (and I don't think you need the return on the next line)

Reply:
Done, thanks